config/hev_config.h

```diff
@@ -0,0 +1,43 @@
+#ifdef CONFIG_TOP
+#include "proffieboard_v2_config.h"
+#define NUM_BLADES 1
+#define NUM_BUTTONS 2
+#define VOLUME 30
+const unsigned int maxLedsPerStrip = 144;
+#define CLASH_THRESHOLD_G 3.0
+#define ENABLE_AUDIO
+#define ENABLE_MOTION
+#define ENABLE_WS2811
+#define ENABLE_SD
+#define DISABLE_NO_REPEAT_RANDOM
```

Comment:
Maybe add a comment as to why you think it is better to disable no repeat random?

Reply:
Comment is good.